applications/openshift/registry/ocp_allowed_registries/rule.yml

```diff
@@ -0,0 +1,28 @@
+prodtype: ocp4
+
+title: "Allowed registries are configured"
+
+description: |-
+    The configuration <tt>registrySources.allowedRegistries</tt> determines the
+    permitted registries that the OpenShift container runtime can access for builds
+    and pods.
+
+rationale: |-
+    Allowed registries should be configured to restrict the registries that the
+    OpenShift container runtime can access.
```

Comment:
It would be great to mention that all other registries are blocked.

Reply:
Documented in 4500b7eeb997bd74f07ecb5bc7c5cfb1e0016d22